docs/api/python/frozen/pyopencolorio_colorspacehelpers.rst

```diff
@@ -3,7 +3,7 @@
   Copyright Contributors to the OpenColorIO Project.
   Do not edit! This file was automatically generated by share/docs/frozendoc.py.
 
-.. py:function:: AddColorSpace(config: PyOpenColorIO.Config, name: str, transformFilePath: str, categories: str = '', connectionColorSpaceName: str) -> None
+.. py:function:: AddColorSpace(config: PyOpenColorIO.PyOpenColorIO.Config, name: str, transformFilePath: str, categories: str = '', connectionColorSpaceName: str) -> None
```

Comment:
Why is this now adding a second PyOpenColorIO for all of these?? Seems fishy to me....?

Reply:
I thought I did the signed commit thing, but I'll redo it.

And that's a good question - I don't know why that happened. I can try rerunning with the `BUILD_FROZEN_DOCS=ON` flag and see what happens.

Reply:
I did it again and nothing changed. Unfortunately the 'frozen' files are only used with BuildTheDocs, and I couldn't get them to work locally without that, so I couldn't QA them.

Reply:
I have built the docs often, but the PyOpenColorIO.PyOpenColorIO I think is due to a change made last year (https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1778) whereby the PyOpenColorIO is now a package (folder) which internally ship the PyOpenColorIO compiled extensions module (as well as other things like command line apps). So this can lead to duplication like here. Is this the first time we re-build python doc since March 2023?

Reply:
It may have been quite a while since we built the docs, yes. We really need to remove the frozen Python docs implementation, which makes it pretty painful to keep everything up to date. For context, the frozen docs generation was implemented to work around the issue of RTD not having an environment for compiling OCIO in order to generate the Python docs. Now that we have OCIO wheels in PyPI, we could instead pip install the latest binary from an RTD build machine and use that for generating docs. I would suggest for now reverting changes made to the frozen docs in this PR, and just contribute updated RST content.

Reply:
I've restored the original frozen rst files